visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.ts

```diff
@@ -36,6 +36,13 @@ export enum ClickType {
 	RightClick = 2
 }
 
+export enum CursorType{
```

Comment:
seems like you're not running the pre-commit hooks. Please run `npm i` on the root and activate `Run Git-Hooks` in IntelliJ. Otherwise, run `npm run format` from the root.